generators/app/index.js

```diff
@@ -1,39 +1,41 @@
-'use strict';
+"use strict";
 
-const Generator = require('yeoman-generator'),
-PromptHandler = require('./resources/handler'),
-handler = new PromptHandler();
+const Generator = require("yeoman-generator"),
+  PromptHandler = require("./resources/handler"),
+  handler = new PromptHandler();
 
 module.exports = Generator.extend({
-
   // Configurations will be loaded here.
   // Ask for user input
   prompting() {
-    return handler.initialPrompt.call(this)
-    .then(answers => handler.secondPrompt.call(this, answers))
-    .then(answers => handler.setPlugins.call(this, answers))
-    .then(answers => handler.thirdPrompt.call(this, answers))
-    .then(answers => handler.finishPrompts.call(this,answers))
-    .then(answers => handler.setNewDestinationRoot.call(this,answers.dir));
+    return handler.initialPrompt
```

Comment:
seeing as we're only supporting node 8.10, let's try and also use async/await instead of `.then`